18-DeviceService/src/PrismSample/ViewModels/MainPageViewModel.cs

```diff
@@ -1,11 +1,22 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using Prism.Mvvm;
+﻿using Prism.Mvvm;
+using Prism.Services;
+using Xamarin.Forms;
 
 namespace PrismSample.ViewModels
 {
     public class MainPageViewModel : BindableBase
     {
+        private readonly IDeviceService _deviceService;
+
+        public string RuntimePlatform { get; set; }
+        public string Idiom { get; set; }
```

Comment:
let's keep these their enum types... also since we're setting these in the ctor we can make them readonly: `public RuntimePlatform RuntimePlatform { get; }`